docs/how-to-guides/themes/block-theme-overview.md

```diff
@@ -98,13 +98,20 @@ Note that it won't take precedence over any of your theme's templates with highe
 
 Some blocks have been made specifically for block themes. For example, you'll most likely use the **Site Title** block in your site's header while your **single** block template will most likely include a **Post Title** and a **Post Content** block.
 
-As we're still early in the process, the number of blocks specifically dedicated to these block templates is relatively small but more will be added as we move forward with the project. As of Gutenberg 8.5, the following blocks are currently available:
+As we're still early in the process, the number of blocks specifically dedicated to these block templates is relatively small but more will be added as we move forward with the project. As of Gutenberg 11.9, the following blocks are currently available:
 
 -   Site Title
+-   Site Logo
+-   Site Tagline
 -   Template Part
 -   Query
 -   Query Loop
 -   Query Pagination
+-   Header
```

Comment:
Header and footer are not new blocks, they are template parts.

Reply:
New proposal in the last commit